auth_server/chart/docker-auth/values.yaml

```diff
@@ -0,0 +1,73 @@
+# Default values for docker-auth.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3
+
+image:
+  repository: cesanta/docker_auth
+  tag: 1.4.0
+  pullPolicy: IfNotPresent
+
+configmap:
```

Comment:
I would say that using the key `configmap` here is confusing. Because some of this data ends up in a Secret and not a ConfigMap. People looking at the values file and not the templates may wonder if you are going to put the tls files into an insecure location. The simple solution may just be to use the word `config` instead which wouldn't have that connotation.

Reply:
Thank you for pointing that. I have split `configmap` into two parts: `configmap` and `secret`, then connotation will disappear.